src/layout/components/GenericCreateUpdateForm/index.vue

```diff
@@ -225,24 +225,9 @@ export default {
       default(error, method, vm) {
         const response = error.response
         const data = response.data
-        if (response.status === 400) {
-          for (const key of Object.keys(data)) {
-            let err = ''
-            let errorTips = data[key]
-            if (errorTips instanceof Array) {
-              errorTips = _.filter(errorTips, (item) => Object.keys(item).length > 0)
-              for (const i of errorTips) {
-                if (i instanceof Object) {
-                  err += i?.port?.join(',')
-                } else {
-                  err += i
-                }
-              }
-            } else {
-              err = errorTips
-            }
-            this.$refs.form.setFieldError(key, err)
-          }
+        if (response.status === 400 && data && typeof data === 'object') {
+          // 覆盖式设置错误映射，避免触发表单内容重建
+          this.$refs.form.setErrors(data)
         }
         this.$emit('performError', data)
       }
```

Comment:
There are no obvious irregularities, potential issues or suggested optimizations in this code snippet. However, I should point out that it has been updated recently so there might not be significant discrepancies between its current version and the one you referenced from 2021-09-01. In general, it seems to have well-written methods and properties with clear goals which is quite standard for a modern JavaScript library. Nonetheless, if performance tuning or readability adjustments could potentially improve its functionality and style further, they may still require careful consideration according to individual coding standards.